.github/workflows/build-images-workflow-run.yml

```diff
@@ -252,6 +252,9 @@ jobs:
             # Run all checks
             ./scripts/ci/selective_ci_checks.sh
           fi
+      - name: Determine GitHub Registry
+        id: determine-github-registry
+        run: ./scripts/ci/images/ci_determine_github_registry.sh
```

Comment:
And then we can remove this step, and the whole new file.

Reply:
Yep. Actually the 'fallback when empty` makes it even simpler.